### ISSUE TITLE
[Request] Remove baselayer dirt from Dark Oak Buildings

### ISSUE THREAD

**Issue by SulpherStaer** [ldtteam/minecolonies#2329]
### Minecolonies version

version 1.12.2-0.8.7809 **ALPHA**

### Expected behavior

- No dirt layer under the 'Roofed Forest type / Dark Oak' buildings, for higher placement freedom & stacking ability

### Actual behaviour

- 'Roofed Forest type / Dark Oak' buildings have a dirt layer under them, and place the tree on a bedding of cobblestone. Remove this dirt & cobblestone layer, so that a greater range of freedom, without ruining the terrain is available.

### Steps to reproduce the problem

1. Place any Dark Oak building (with exception of the Medival Dark Oak) and observe dirt layer, which ruins terrain and prevents stacking.
![2018-03-20_20 19 58](https://user-images.githubusercontent.com/5037244/37677865-9923ae84-2c7c-11e8-8271-f814e1e0ac40.png)

**Comment by Kisonu** [comment]
i imagine cobblestone under the tree may be what prevents the lumbermen from recognizing the building as a "tree".. most of the barracks have this same style.. dirt may be unnecessary though and i'd agree with removing that.

**Comment by PoroUsedSnax** [comment]
I believe this is already fixed, just that the style needs rescanning, which is a lot of buildings, it will come eventually:tm: along with a rescan of a few other styles.

**Comment by lunadragnstar** [comment]
I am actually working on this now, and I created new "buildings" for the composter, smeltery, and the herder huts. Hope to finish today and do some testing.

**Comment by lunadragnstar** [comment]
Ok, so I am finished. I recreated all of the original buildings and only tweaking a few of them. I changed the restaurant a bit because I didn't like all the unprotected edges, I changed the Lumberjack because he wouldn't cut down the trees on the platform so I just took that out and put in stairs. I put the tree house field in a tree. I added buildings for the composter, the smeltery and the herders. I fixed the barracks. And I added floating stairs. I also made it so that the buildings disturb as little of the surroundings as possible because in the Dark Oak areas trees are everywhere and the point of building in there is camouflage. (This may cause you to need to clean up some areas around your build site but I didn't want big squares of empty space in the middle of the dark oak forest. 
[Dark Oak Alt.zip](https://github.com/ldtteam/minecolonies/files/2687598/Dark.Oak.Alt.zip)
